Cargo.toml

```diff
@@ -18,5 +24,6 @@ termcolor = "1.1.0"
 regex = "1.3.9"
 
 [dev-dependencies]
+clap = "2.33.3"
```

Comment:
It’s just another thing to keep in-sync. I’d think it’d be better if the example was just a minimal example of integrating with deno_doc

Reply:
This is a dev dependency. It doesn't need to be kept in sync. Just like for `deno_lint`.

Reply:
This is my first time using Clap, and it is indeed a good start for creating cli applications.  But I have no idea whether to recommend it to others as an example. This is a trivial matter and users will not be confused by it.